extensions/amp-video/validator-amp-video.protoascii

```diff
@@ -56,6 +56,10 @@ tags: {  # <amp-video>
   attrs: { name: "placeholder" }
   attrs: { name: "poster" }
   attrs: {
+    name: "crossorigin"
+    value_regex: "(^$|anonymous|use-credentials)"
```

Comment:
Cool, can do. I want to double check though, since in the original issue you said

> I think we can just expose crossorigin as an attribute on amp-video (requires validation rule changes) and propagate it to the underlying video